### PR TITLE
Force fragment size histogram file to exist in rnaseqc output

### PR DESCRIPTION
**Pull request by kachulis**
If there are no fragments used to generate fragment size histogram, rnaseqc does not generate a fragment size histogram file, which causes the task to fail.  This PR forces an empty file to be produced in this case, so that the task succeeds.

### CODE REVIEW

**Review by mmorgantaylor — APPROVED** [review]

**Comment by jessicaway** [comment]
add to whitelist

**Review by jessicaway — APPROVED** [review]

File: pipelines/broad/rna_seq/RNAWithUMIsPipeline.changelog.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+#1.0.10
```
Comment (jessicaway) [inline pipelines/broad/rna_seq/RNAWithUMIsPipeline.changelog.md:1]: you need to update the `pipeline_version` variable in the WDL as well:
https://github.com/broadinstitute/warp/blob/develop/pipelines/broad/rna_seq/RNAWithUMIsPipeline.wdl#L23

**Comment by jessicaway** [comment]
`BroadInternalRNAWithUMIs.changelog.md` also needs to be updated. (No need to increment the version, just add a bullet point to the notes for the most recent version)

**Comment by jessicaway** [comment]
retest this please

**Comment by jessicaway** [comment]
you can ignore the failing trivy check, it is not related to these changes

**Pull request merged**